conan/tools/env/environment.py

```diff
@@ -0,0 +1,230 @@
+import fnmatch
+import os
+import textwrap
+from collections import OrderedDict
+
+from conans.errors import ConanException
+from conans.util.files import save
+
+_ENV_VAR_PLACEHOLDER = "$PREVIOUS_ENV_VAR_VALUE%"
```

Comment:
should be named "CONAN_SOMETHING" like the `_PATHSEP`?

Reply:
Fixed: I have replaced those strings for real types, to be able to filter the list of values more reliably